CMakeLists.txt

```diff
@@ -100,15 +102,14 @@ if(OPENSSL_FOUND)
         ${OPENSSL_INCLUDE_DIR}
     )
 
-    set_property(TARGET sioclient_tls PROPERTY CXX_STANDARD 11)
-    set_property(TARGET sioclient_tls PROPERTY CXX_STANDARD_REQUIRED ON)
-
+    targeet_compile_features(sioclient_tls PUBLIC cxx_std_11)
```

Comment:
```suggestion
    target_compile_features(sioclient_tls PUBLIC cxx_std_11)
```

Reply:
@jmigual Thank you for checking! Shame I didn't see it first.